src/main/java/com/adyen/model/checkout/PaymentsRequest.java

```diff
@@ -185,6 +185,9 @@
     @SerializedName("storePaymentMethod")
     private Boolean storePaymentMethod = null;
 
+    @SerializedName("store")
+    private String store = null;
```

Comment:
Please update equals()  method to add this field

Reply:
Also updated !